avae/evaluate.py

```diff
@@ -65,15 +94,50 @@ def evaluate(datapath, lim, splt, batch_s, collect_meta, use_gpu):
     if config.VIS_REC:
         vis.recon_plot(x, x_hat, name="evl")
 
+    # get training latent space
+    latents_training = meta_df[meta_df["mode"] == "trn"][
```

Comment:
Should be reading latent space from `.pt` file and not `.pkl` file, will fix this before merging.